install-and-configure/install/helm-install-params.md

```diff
@@ -10,9 +10,9 @@ For example, you can only pass a product key if that is all you need to configur
 $ helm install kubecost cost-analyzer \
     --repo https://kubecost.github.io/cost-analyzer/ \
     --namespace kubecost --create-namespace \
-    --set kubecostProductConfigs.productKey.key="123"
+    --set kubecostProductConfigs.productKey.key="123" \
+    --set kubecostProductConfigs.productKey.enabled=true
     ...
-```
```

Comment:
Looks like the suggestion removed the closing back ticks.

Reply:
Fixed